### PR TITLE
fix type of MessageData message field

### PR DESCRIPTION
**Pull request by Seb-figment**

### CODE REVIEW

File: sf/cosmos/type/v1/type.proto [hunk 1]
```diff
@@ -40,7 +40,7 @@ message TransactionData {
 }
 
 message MessageData {
-  Any message = 1;
+  google.protobuf.Any message = 1;
```
Comment (pienkowb) [inline sf/cosmos/type/v1/type.proto:43]: Any reason we need the original `Any` type here? `MessageData` is created on the Graph Node side and we can pass our custom `Any` type there.

Comment (Seb-figment) [reply sf/cosmos/type/v1/type.proto:43]: It's because of the generated AscMessageData object from the type.proto. To keep everything in sync.

Comment (Seb-figment) [reply sf/cosmos/type/v1/type.proto:43]: Also don't have to create new object from 
```
message TxBody {
  repeated google.protobuf.Any messages = 1;
...
```

**Review by patiee — APPROVED** [review]
Looks good

**Pull request merged**